stable/elasticsearch-curator/values.yaml

```diff
@@ -101,5 +101,40 @@ priorityClassName: ""
 #     mountPath: /certs
 #     readOnly: true
 
+# Add your own init container or uncomment and modify the given example.
+extraInitContainers: {}
```

Comment:
can you please add a test to exercise this option? https://github.com/helm/charts/tree/master/test#providing-custom-test-values

the test init container can be very dummy like just echo a text

Reply:
Have added this improvement as requested.